foreman.te

```diff
@@ -591,14 +604,15 @@ optional_policy(`
     tunable_policy(`passenger_run_foreman', `
         files_manage_generic_tmp_files(passenger_t)
         files_manage_generic_tmp_dirs(passenger_t)
+        logging_search_logs(passenger_t)
+        logging_list_logs(passenger_t)
     ')
 ')
 optional_policy(`
     tunable_policy(`passenger_run_foreman', `
         manage_files_pattern(passenger_t, system_cronjob_tmp_t, system_cronjob_tmp_t)
         manage_dirs_pattern(passenger_t, system_cronjob_tmp_t, system_cronjob_tmp_t)
-        read_files_pattern(httpd_t, puppet_etc_t, puppet_etc_t)
-        read_lnk_files_pattern(httpd_t, puppet_etc_t, puppet_etc_t)
+        puppet_read_config(httpd_t)
```

Comment:
Isn't this duplicate from line 552 where it's unconditionally allowed?

Reply:
Yes, we actually made copies of all rules into `passenger_run_foreman` blocks (for technical reason it can't be just a single block). This was done because we shipped versions with passenger and we were not sure about if we are fully dropping it.

Once you guys confirm that Passenger is gone for good, I will be more than happy remove all of these rules and the boolean flag as well. :-)

Reply:
I still don't understand why we effectively have:
```
 # Apache httpd reads puppet certs
optional_policy(`
    puppet_read_config(httpd_t)
') 

optional_policy(`
    tunable_policy(`passenger_run_foreman', `
        puppet_read_config(httpd_t)
    ')
')
```

Isn't this the same sufficient for both Puma and Passenger setups?
```
 # Apache httpd reads puppet certs
optional_policy(`
    puppet_read_config(httpd_t)
') 
```